app/components/Task.tsx

```diff
@@ -105,12 +106,12 @@ const Task = () => {
                       value={task}
                       onChange={(e) => updateTask(index, e.target.value)}
                       placeholder='Add details here'
-                      className='border-2 p-1'
+                      className='border-2 p-1 w-10/12'
                     />
                     {/* DELETE TASK */}
                     <button
                       onClick={() => deleteItem(index)}
-                      className='p-1 size-8 rounded-md bg-neutral-500'
+                      className='p-1 size-8 rounded-md bg-red-500'
```

Comment:
i see that every button you have got p-1. Meaning you can style it arah globals.css

in globals.css
```
button {
    @apply p-1;
}
```

you can add other styling also if you see yourself keep repeating some styles

Reply:
noted. For now still focusing on the function and the design is not yet decided.

Reply:
ok